uninstall.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -e
+
+uninstall_logid() {
+    if [ 1 -gt ${#1} ]; then
+        printf '%s\n' "No manifest_path detected for uninstall. Skipping..."
+        return
+    fi
+    local manifest_path="$1"
+    echo "manifest_path is $manifest_path"
+    while IFS= read -r line; do 
+        printf '%s\n' "Removing $line.."
+        rm "$line"
+    done < <(cat "$manifest_path")
+
+    printf '%s\n' "Uninstall complete"
+}
+
+get_manifest_path() {
+    local manifest_path=$(find . -name 'install-manifest.txt' -type f -print)
```

Comment:
The file is named `install_manifest.txt` , not `install-manifest.txt` , at least when I tried it.